scripts/BUILD.bazel

```diff
@@ -77,3 +78,22 @@ kt_jvm_binary(
     main_class = "org.oppia.android.scripts.xml.XmlSyntaxCheckKt",
     runtime_deps = ["//scripts/src/java/org/oppia/android/scripts/xml:xml_syntax_check_lib"],
 )
+
+TEST_FILE_EXEMPTION_ASSETS = generate_test_file_assets_list_from_text_protos(
+    name = "test_file_exemption_assets",
+    test_file_exemptions_name = ["test_file_exemptions"],
+)
+
+kt_jvm_library(
```

Comment:
This should also be marked as testonly. Ditto for earlier PRs if it applies.

Reply:
Done.